README.md

```diff
@@ -1 +1,15 @@
-# ens-validation
+# ENS Validation
+
+Following the alorithm used for Google's IDN policy, this library can validate a ENS domain.
+
+## Test
+
+```sh
+yarn test
+```
+
+## Build
+
+```sh
+yarn build
+```
```

Comment:
Perhaps we could add a little section about how to contribute additional examples to domains.json.